lib/buffer.js

```diff
@@ -379,6 +378,16 @@ Buffer.prototype.toString = function(encoding, start, end) {
 };
```

Comment:
FWIW, I included the style fix in the final commit.  It didn't make the diff any noisier.